infra/src/webapp.tf

```diff
@@ -156,10 +156,10 @@ locals {
     JIRA_ORGANIZATION_NAME_CUSTOM_FIELD = var.jira_organization_name_custom_field
 
     # Apim connection
-    AZURE_APIM                                   = var.azure_apim
-    AZURE_APIM_RESOURCE_GROUP                    = var.azure_apim_resource_group
-    AZURE_APIM_DEFAULT_SUBSCRIPTION_PRODUCT_NAME = "io-services-api" # FIXME: create and use variable
-    AZURE_SUBSCRIPTION_ID                        = data.azurerm_subscription.current.subscription_id
+    AZURE_APIM                           = var.azure_apim
+    AZURE_APIM_RESOURCE_GROUP            = var.azure_apim_resource_group
+    AZURE_APIM_SUBSCRIPTION_PRODUCT_NAME = "io-services-api" # FIXME: create and use variable
```

Comment:
TIP: Since you are editing this part it would be a good thing to solve the FIXME by creating a proper variable